src/conn/conn_log.c

```diff
@@ -252,6 +267,7 @@ __logmgr_config(WT_SESSION_IMPL *session, const char **cfg, bool *runp, bool rec
     if (!*runp)
         return (0);
 
+    __wt_logmgr_compat_version(session);
```

Comment:
Does it make sense to still have this call here,  when it has already been done in `__wt_conn_compat_config`, earlier in the process?  Or am I missing something about the timing?